pio/pio.go

```diff
@@ -157,7 +157,7 @@ func GetConfigPath() (p string, err error) {
 	return
 }
 
-// GetFilesDir is used to get the directory that we store
+// GetEncryptedFilesDir is used to get the directory that we store
```

Comment:
If you’d like, this can be improved further:

```
// GetEncryptedFilesDir returns the directory that ...
```

Because it’s preferable to be more direct when describing what a function does. Optional, of course.

I realize that many other funcs in this package have a similar “Foo is used to ...” phrasing, so it may not be worth changing just one. Your call.